src/AsyncEventSource.h

```diff
@@ -96,6 +98,7 @@ class AsyncEventSourceClient {
     AsyncClient* client(){ return _client; }
     void close();
     void write(const char * message, size_t len);
+    bool try_write(const char * message, size_t len);
```

Comment:
why overloading ? in both cases, there is no send guarantee. 
I would just return false bool on the original method if the idea is to get a feedback whether the message was enqueued or not

Reply:
For backwards compatibility.  There's an argument to be made that the caller can just ignore the bool and that since the code is being recompiled anyway it's not an ABI issue, but it's just general principle.

Reply:
you're right. It can break if someone is doing something like: `return send(...);` where `send()` initially returns void and is now changed to return bool. 

But for this use case, I would say that this is debatable... Should the laziness of the dev who write that instead of adding a `return;` in the next line be considered... At the end, when someone writes that, it specifically binds the code to a method signature, when this is not needed, and then locks / break on further improvements done in the api.